spec_mode/mirimrs/notebook1/MRS_Notebook1.ipynb

```diff
@@ -24,7 +24,7 @@
    "source": [
```

Comment:
<p>This cell can probably go now that you've added the <code>jwst</code> version check earlier on in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>